internal/sqld/sqldwebsockets/websockets.go

```diff
@@ -0,0 +1,230 @@
+package sqldwebsockets
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"strconv"
+
+	"nhooyr.io/websocket"
+	"nhooyr.io/websocket/wsjson"
+	"vitess.io/vitess/go/pools"
+)
+
+func errorMsg(errorResp interface{}) string {
+	return errorResp.(map[string]interface{})["error"].(map[string]interface{})["message"].(string)
+}
+
+func isErrorResp(resp interface{}) bool {
+	return resp.(map[string]interface{})["type"] == "response_error"
+}
+
+type SqldWebsocket struct {
```

Comment:
The `Sqld` prefix is redundant, we are already in a sqld-specific package:
https://go.dev/doc/effective_go#names

Reply:
I didn't find anything relevant under the link you pasted. Could you please point me to the exact sentence/paragraph you wanted me to read that supports your claim?

Reply:
> The importer of a package will use the name to refer to its contents, so exported names in the package can use that fact to avoid repetition. For instance, the buffered reader type in the bufio package is called Reader, not BufReader, because users see it as bufio.Reader, which is a clear, concise name.

Reply:
fixed.